<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Геометрия</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="rContact">
           <property name="text">
            <string>Радиус контактов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="siezeBlocks">
           <property name="text">
            <string>Размер блоков</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="widthLines">
           <property name="text">
            <string>Тощина линий элементов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="widthLineContatcts">
           <property name="text">
            <string>Толщина линий контактов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="greedButton">
           <property name="text">
            <string>Изменить шаг сетки</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Панель</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="deleteConnection">
           <property name="text">
            <string>Удалить соединение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addBlock">
           <property name="text">
            <string>Добавить блок</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPoint">
           <property name="text">
            <string>Добавить точку</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPolus">
           <property name="text">
            <string>Добавить полюс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addRelay">
           <property name="text">
            <string>Добавить реле</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addContactRelay">
           <property name="text">
            <string>Добавить контакт реле</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="sceneLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
